lstchain/reco/dl0_to_dl1.py

```diff
@@ -354,8 +357,12 @@ def r0_to_dl1(input_filename=get_dataset_path('gamma_test_large.simtel.gz'),
 
                     writer.write(table_name=f'telescope/image/{tel_name}',
                                  containers=[event.r0, tel, extra_im])
-                    writer.write(table_name=f'telescope/parameters/{tel_name}',
-                                 containers=[dl1_container])
+
+                    if config['tailcut']['picture_thresh'] > 0 \
```

Comment:
Just re-checking, if the parameter table is not written here, the code would break when using simulations and trying to access the parameter table to write the disp, right?
https://github.com/cta-observatory/cta-lstchain/blob/b330595571dd0becb5fbc243b4ce6b4b85197bbd/lstchain/reco/dl0_to_dl1.py#L373

Reply:
You are right @rlopezcoto, actually, I did not test the script with MC data. Does it make sense to include also the condition `and not is_simu` to the others one as a workaround?

Reply:
I would say that in the case of simulations we may also want to write all events for some reason, so you could include the same `if` condition to write the disp information in the parameter table only in case some cleaning > 0 is given